### ISSUE TITLE
[BUG] chat embed_chatflow config_sessionId not working

### ISSUE THREAD

**Issue by Kvkthecreator** [FlowiseAI/Flowise#3587]
**Describe the bug**
when trying to use the html embed for chat (both widget, and fullpage html)., 
i am sending the chatflow config additional information
- specifically sessionId, is not working. 
- the conversation doesn't show the conversation specific to referenced sessionId

**To Reproduce**

test html embed chat, both full page and widget.
- chatflow config with sessionId. 
- see if only sessionId specific information is shown. 

**Expected behavior**
html embed chat should show only the message history of specified sessionId.
<img width="1152" alt="image" src="https://github.com/user-attachments/assets/506a3eec-cc86-4294-ae13-e2b7c9cfbde4">

[1120_SNSmktgAgent Agents.json](https://github.com/user-attachments/files/17931780/1120_SNSmktgAgent.Agents.json)
**Flow**
If applicable, add exported flow in order to help replicating the problem.

**Setup**

-   Installation [render, bubble]
-   Flowise Version [latest 11.27 latest versions deployed]
-   OS: [e.g. macOS]
-   Browser [safari]

**Additional context**

**Comment by SirLucian** [comment]
This is a feature not a bug to do with override variables.

You now have to configure them in your global variables, you can't just declare them on the fly. so I add a couple empty objects for things I want to declare in api calls and then in my chatflow config I can toggle them to allow for overwriting

**Comment by Kvkthecreator** [comment]
thanks for the reply... may i ask are you sure you're not referring to override config for APIs? (b/c yes, i am also using global vars. here)
- i'm running into trouble specifically for chat embed. (am looking at flowise doc, but doesn't distinctly specify that vars can be used for chat embed config)
- that being said, i don't mind testing your approach.
- how do you call the vars for the below? i'm looking to changed the sessionId.

<script type="module">
  import Chatbot from 'https://cdn.jsdelivr.net/npm/flowise-embed/dist/web.js';
  Chatbot.init({
    chatflowid: 'abc',
    apiHost: 'http://localhost:3000',
    chatflowConfig: {
      "sessionId": "123",
      "returnSourceDocuments": true
    }
  })
</script>

**Comment by HenryHengZJ** [comment]
I dont think we are able to do so since the beginning, you can only set a sessionId that any chat conversation happened on the chat, will be saving to that sessionid, however you are not able to display the chat by specifying a specific sessionId for security purpose

**Comment by Kvkthecreator** [comment]
thanks alot for your comment. 
if that is the case, is there anyway i can:
- use the same agentflow, and use the chat embed (or if i must API), and have multiple users have their own "conversation session" or "conversation threads?

**Comment by pgaurav007** [comment]
Please answer this @HenryHengZJ , otherwise embed chat is useless: 
Description:
After logging out and logging in as a different user, the embedded Flowise chat widget continues the previous user's chat session. The session context is not refreshed, causing the second user to see the first user's conversation history.

Root Cause:
The Flowise chat widget retains the existing session ID across login/logout because a new session ID is not generated when initializing the chat for the new user.

Expected Behavior:
Each time a user logs in, a fresh session ID should be generated and passed to the Flowise widget to start a new isolated chat session

Even though i overrite it, it still shows previous chats.
